meetups/2020/jan/25/sponsor_pitch.md

```diff
@@ -0,0 +1,94 @@
+# Code-n-build
+
+We, DSC JECRC University are truly excited to host our first big event on the 24th and 25th of January!
+
+We have our preparations at a full swing with each lead at work.  Before going any further let's look into what it's all about!
+
+https://codenbuild.tech/
+
+## About
+
+We are bringing out the first edition of "CodeNBuild - who let the codes out", the biggest hackathon JECRC has ever witnessed.
+
+We bring a widely known hackathon culture with our own added fun element.
+
+We intend to make this event as successful as last year’s local hackathon while taking a step forward and bringing together passionate participants to compete from states across India.
+
+We are expecting 500 - 600 participants from all over India. It is compulsory to have the 6th member as Girl if team size exceeds more than 5 and the minimum team size will be 4 members.
```

Comment:
This should be in the participant pitch no?

Reply:
yes